qa/rpc-tests/wallet_sapling.py

```diff
@@ -201,5 +201,63 @@ def run_test(self):
         except JSONRPCException as e:
             assert_equal("Cannot send to both Sprout and Sapling addresses using z_sendmany", e.error['message'])
 
+        # Check z_sendmanywithchangetosender
+        # send from node 0 taddr to node 2 single taddr
+        senderaddr = self.nodes[0].getnewaddress()
+        senderaddr = self.nodes[0].getnewaddress()
+        self.nodes[0].generate(10)
+        self.sync_all()
+
+        senderExpectedBalance = Decimal('100.0')
+        mytxid = self.nodes[0].sendtoaddress(senderaddr, senderExpectedBalance)
+        self.sync_all()
+        self.nodes[0].generate(1)
+        self.sync_all()
+        assert_equal(self.nodes[0].z_getbalance(senderaddr), senderExpectedBalance)
+
+        receiveraddr = self.nodes[2].getnewaddress()
+        receiveraddr = self.nodes[2].getnewaddress()
```

Comment:
why getting receiveraddr is repeated 2 times

Reply:
fixed